ChronoVisor/src/VisorClientPortal.cpp

```diff
@@ -253,18 +248,12 @@ int chronolog::VisorClientPortal::ReleaseStory(chl::ClientId const &client_id, s
     { return CL_ERR_NOT_AUTHORIZED; }
 
     StoryId story_id(0);
-    bool notify_keepers = false;
-    auto return_code = chronicleMetaDirectory.release_story(client_id, chronicle_name, story_name, story_id
-                                                            , notify_keepers);
+    auto return_code = chronicleMetaDirectory.release_story(client_id, chronicle_name, story_name, story_id);
     if(chronolog::CL_SUCCESS != return_code)
     { return return_code; }
 
-    if(notify_keepers && theKeeperRegistry->is_running())
-    {
-        std::vector <chronolog::KeeperIdCard> recording_keepers;
-        theKeeperRegistry->notifyKeepersOfStoryRecordingStop(theKeeperRegistry->getActiveKeepers(recording_keepers)
-                                                             , story_id);
-    }
+    theKeeperRegistry->notifyRecordingGroupOfStoryRecordingStop(story_id);
+
     return chronolog::CL_SUCCESS;
```

Comment:
TODO: we need a `chronolog::ReleaseStoryResponseMsg` to return error code to clients.